samples/CloudClipboard/README.md

```diff
@@ -0,0 +1,16 @@
+# Azure SDK for .NET Samples
```

Comment:
This needs to be fleshed out and should reflect the sample.    Please see [this](https://github.com/Azure/azure-sdk-for-net/tree/master/samples/iothub-connect-to-eventhubs) for inspiration.

Reply:
Updated